### PR TITLE
Add installation for Windows

### PR DESCRIPTION
**Pull request by ignatovich-artem**
Add installation for Windows: https://github.com/lean-delivery/ansible-role-zookeeper/issues/10
Fixed: https://github.com/lean-delivery/ansible-role-zookeeper/issues/12
Known issues: https://github.com/lean-delivery/ansible-role-zookeeper/issues/14 https://github.com/lean-delivery/ansible-role-zookeeper/issues/15 https://github.com/lean-delivery/ansible-role-zookeeper/issues/16

### CODE REVIEW

File: tasks/Linux/system.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  include_tasks: '{{ zk_install_type }}.yml'
+
+- name: Enable zookeeper service
+  service:
+    name: '{{ zk_service_name }}'
+    enabled: true
+  register: dbg
+  become: true
+  when:
+    - zk_service_autostart
```
Comment (tgadiev) [inline tasks/Linux/system.yml]:
```suggestion
    - zk_service_autostart | bool
```

File: tasks/Linux/system.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+- name: Zookeeper Installation
+  include_tasks: '{{ zk_install_type }}.yml'
+
+- name: Enable zookeeper service
+  service:
+    name: '{{ zk_service_name }}'
+    enabled: true
+  register: dbg
```
Comment (tgadiev) [inline tasks/Linux/system.yml]: Why do we need this variable?

File: molecule/resources/tests/verify.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+- name: Verify Linux
+  hosts: rhel_family:debian_family
+  tasks:
+  - name: Check Zookeeper service status
+    service:
+      name: 'zookeeper'
+    check_mode: true
+    register: zookeeper_status
```
Comment (tgadiev) [inline molecule/resources/tests/verify.yml]: Why we don't use simple `service_facts` module?

File: tasks/Linux/system.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  become: true
+  when:
+    - zk_service_autostart
+
+- name: Check Zookeeper service status
```
Comment (tgadiev) [inline tasks/Linux/system.yml]: Why do we need this step?

File: tasks/Win32NT/choco_install.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+
+- name: Install zookeeper from chocolatey
+  win_chocolatey:
+    name: apache-zookeeper
+    package_params: "unzipLocation=C:/zookeeper/"
```
Comment (tgadiev) [inline tasks/Win32NT/choco_install.yml]: Why hardcoding the path?

File: README.md [hunk 6]
```diff
@@ -23,114 +29,95 @@ Requirements
...
-
-    default: `/var/lib/zookeeper`
+    default: `'{{ zk_default_data_dir }}'`
 
  - `zk_log_dir` - logs directory
-
-    default: `/var/log/zookeeper`
+    default: `{{ zk_default_log_dir }}`
 
  - `zk_dir` - zookeeper direcotry
```
Comment (tgadiev) [inline README.md]:
```suggestion
 - `zk_dir` - zookeeper directory
```

File: tasks/Win32NT/tarball_install.yml [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+      return_content: true
+    register: artifact_checksum_file
+
+  - name: Debug artifact_checksum_file
+    debug:
+      var: artifact_checksum_file
+
+  - name: Get artifact checksum
+    set_fact:
+      artifact_checksum: '{{ artifact_checksum_file.content.split(" ")[0] }}'
```
Comment (tgadiev) [inline tasks/Win32NT/tarball_install.yml]:
```suggestion
      artifact_checksum: '{{ artifact_checksum_file.content | trim }}'
```

File: tasks/Linux/system.yml (hunk 1 above)
Comment (ignatovich-artem) [reply tasks/Linux/system.yml]: will be fixed

File: tasks/Linux/system.yml (hunk 2 above)
Comment (ignatovich-artem) [reply tasks/Linux/system.yml]: Who knows, i'll look into this.

File: tasks/Linux/system.yml (hunk 4 above)
Comment (ignatovich-artem) [reply tasks/Linux/system.yml]: Who knows, i'll look into this.

File: tasks/Win32NT/choco_install.yml (hunk 5 above)
Comment (ignatovich-artem) [reply tasks/Win32NT/choco_install.yml]: I think this can be improved.

File: README.md (hunk 6 above)
Comment (ignatovich-artem) [reply README.md]: Will be fixed

File: tasks/Win32NT/tarball_install.yml (hunk 7 above)
Comment (ignatovich-artem) [reply tasks/Win32NT/tarball_install.yml]: Iam not trying to remove trailing whitespaces, I need to get a first part of a string:
b14f7a0fece8bd34c7fffa46039e563ac5367607c612517aa7bd37306afbd1cd  zookeeper-3.4.14.tar.gz

File: molecule/resources/tests/verify.yml (hunk 3 above)
Comment (ignatovich-artem) [reply molecule/resources/tests/verify.yml]: There are several reasons for this:
1. I want to stick to the same approach for Linux and Windows, i.e. **service** module for Linux and **win_service** for Windows.
2.  **service_facts** gives not very accurate results, for example, in the case described in this issue https://github.com/lean-delivery/ansible-role-zookeeper/issues/16 **service_facts** reports that service is running, but it is not.

**Comment by tgadiev** [comment]
Done by subsequent: https://github.com/lean-delivery/ansible-role-zookeeper/pull/18

**Pull request closed without merging**